src/FINDOLOGIC/Export/CSV/CSVItem.php

```diff
@@ -2,49 +2,51 @@
 
 namespace FINDOLOGIC\Export\CSV;
 
+use BadMethodCallException;
+use DOMDocument;
 use FINDOLOGIC\Export\Data\Attribute;
+use FINDOLOGIC\Export\Data\Image;
 use FINDOLOGIC\Export\Data\Item;
 use FINDOLOGIC\Export\Data\Usergroup;
 use FINDOLOGIC\Export\Helpers\DataHelper;
+use InvalidArgumentException;
 
 class CSVItem extends Item
 {
     /**
      * @inheritdoc
      */
-    public function getDomSubtree(\DOMDocument $document): void
+    public function getDomSubtree(DOMDocument $document): void
     {
-        throw new \BadMethodCallException('CSVItem does not implement XML export.');
+        throw new BadMethodCallException('CSVItem does not implement XML export.');
     }
 
     /**
      * @inheritdoc
      */
     public function getCsvFragment(array $availableProperties = []): string
     {
-        $that = $this; // Used in closure.
-
         $id = $this->getId();
-        $ordernumbers = $this->sanitize($this->ordernumbers->getCsvFragment());
-        $name = $this->sanitize($this->name->getCsvFragment());
-        $summary = $this->sanitize($this->summary->getCsvFragment());
-        $description = $this->sanitize($this->description->getCsvFragment());
+        $ordernumbers = CSVItem::sanitize($this->ordernumbers->getCsvFragment());
```

Comment:
Use `self::sanitze(` if we stick to a static method here. This applies to all changes in the file